RELEASE_NOTES

```diff
@@ -22,26 +20,9 @@ could be different.
 New Features
 ------------
 
-support M1 ARM architecture
+Basic SQL support
 
 Incompatible changes
 --------------------
 
 - NONE -
-
```

Comment:
Are we going to remove these lines?

Reply:
Ya, since I didn't found anything in our current release notes. When I missed something please add :)